app/players/dwarves/dwarf.py

```diff
@@ -0,0 +1,11 @@
+from typing import NoReturn
+from ..player import Player

```

Comment:
fix here and other places 

```suggestion
from app.players.player import Player

```